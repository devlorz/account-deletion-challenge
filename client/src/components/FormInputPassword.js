import PropTypes from 'prop-types'
import React from 'react'

const FormInputPassword = props => {
  const { onTypeEmail, errorMessage } = props
  return (
    <div>
      <input
        type="text"
        placeholder="ross@example.com"
        style={{ width: '350px' }}
        onChange={onTypeEmail}
      />
      <span style={{ color: 'red' }}>{errorMessage}</span>
    </div>
  )
}

FormInputPassword.PropTypes = {
  onTypeEmail: PropTypes.func,
  errorMessage: PropTypes.string,
}

export default FormInputPassword
