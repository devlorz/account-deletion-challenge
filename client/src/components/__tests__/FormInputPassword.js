import React from 'react'
import { render } from 'react-testing-library'

import FormInputPassword from '../FormInputPassword'

test('snapshot', () => {
  const { container } = render(
    <FormInputPassword onTypeEmail={jest.fn()} errorMessage={''} />
  )
  expect(container.firstChild).toMatchSnapshot()
})
