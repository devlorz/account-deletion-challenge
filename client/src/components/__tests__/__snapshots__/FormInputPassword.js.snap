// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<div
  data-reactroot=""
>
  <input
    placeholder="ross@example.com"
    style="width: 350px;"
    type="text"
  />
  <span
    style="color: red;"
  />
</div>
`;
